app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Same for the Signup page, it should be modified to match the wireframes provided, kindly consider implementing this so your web app looks like the wireframes. you can style the app however you like but it should follow the wireframe (you can center the input fields and give them some padding, also there is no label kindly use placeholders instead. The button should be centered and take the whole width to match the input fields) 

|Your app|Wireframe|
|-|-|
|![Screenshot_20230727_112755](https://github.com/christianonoh/beyond-borders/assets/25227022/3d9832bc-1e6f-463f-991d-e2f1e7cb7d3c)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)|